src/backend/tests/reimbursement-requests.test.ts

```diff
@@ -625,4 +626,28 @@ describe('Reimbursement Requests', () => {
       expect(newReimbursement).toStrictEqual(reimbursementTransformer(reimbursementMock));
     });
   });
+
+  describe('Edit Vendor Tests', () => {
+    test('Throws error if user isnt an admin', async () => {
```

Comment:
Let's update this test name after we update the permission in the service function.